### PR TITLE
New feedstock: dbus-cxx

### PR DESCRIPTION
**Pull request by pkgw**
These are C++ bindings for DBus.

### CODE REVIEW

**Comment by pkgw** [comment]
Took a few tries, but it works!

File: recipes/dbus-cxx/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,55 @@
+# Note that while Conda accepts a package name of "dbus-c++", Anaconda.org
+# rejects it.
```
Comment (scopatz) [inline recipes/dbus-cxx/meta.yaml]: Might be worth reporting this to Continuum.

File: recipes/dbus-cxx/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    - glib
+
+test:
+  commands:
+    - ls $PREFIX/lib/libdbus-c++-glib-1.so  # [linux]
+    - ls $PREFIX/lib/libdbus-c++-glib-1.dylib  # [osx]
+
+about:
+  home: http://dbus-cplusplus.sourceforge.net/
+  license: LGPLv2+
```
Comment (scopatz) [inline recipes/dbus-cxx/meta.yaml]: Please include the `license_file`.  Since this is LGPL it is required.

**Comment by scopatz** [comment]
Just a couple of small issues, but looks good.

**Comment by pkgw** [comment]
After the latest update I think this should be good to merge.

**Comment by scopatz** [comment]
Thanks!

**Pull request merged**